inference-engine/src/vpu/graph_transformer/src/frontend/detect_network_batch.cpp

```diff
@@ -196,12 +154,12 @@ void FrontEnd::detectNetworkBatch(
 
         const auto origShape = outputShapes[p.first];
         const auto newShape = ieData->getDims();
-
+        VPU_THROW_UNLESS(origShape != newShape, "Have unbatched output");
         if (origShape == newShape) {
```

Comment:
I think this check is not required, given that we are checking the inequality at line 157

Reply:
This line was added for trying to find real use cases by using dlbenchmark.
The PR will have to be reworked again after the validation.